common/src/main/java/io/netty/util/internal/OneTimeTask.java

```diff
@@ -15,14 +15,11 @@
  */
 package io.netty.util.internal;
 
-import io.netty.util.concurrent.EventExecutor;
 
 /**
- * {@link Runnable} which represent a one time task which may allow the {@link EventExecutor} to reduce the amount of
- * produced garbage when queue it for execution.
- *
- * <strong>It is important this will not be reused. After submitted it is not allowed to get submitted again!</strong>
+ * @deprecated Use {@link Runnable}.
  */
+@Deprecated
 public abstract class OneTimeTask extends MpscLinkedQueueNode<Runnable> implements Runnable {
```

Comment:
This class is in an internal package. My vote is to remove it if we no longer need it. Lets try to keep the code base lean (for Netty :wink:) and mean where possible. Also using `OneTimeTask` will no longer provide the memory savings which is was most likely used for.